examples/glusterfs/README.md

```diff
@@ -0,0 +1,42 @@
+## Glusterfs
+
+[Glusterfs](http://www.gluster.org) is an open source scale-out filesystem. These examples provide information about how to allow Docker containers use Glusterfs volumes.
+
+The example consists of a pod that runs on hosts that install Glusterfs client package.
```

Comment:
nit: Rephrase as 'The example assumes that the Glusterfs client package i installed on all nodes'.